clients/admin-ui/src/features/common/Stepper.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Stack, Text } from "@fidesui/react";
+import React from "react";
+
+import steps from "../config-wizard/constants";
+import { StepperCircleIcon, VerticalLineIcon } from "./Icon";
+
+interface Props {
+  activeStep: number | null;
+}
+
+const Stepper = ({ activeStep }: Props) => (
+    <Stack direction={["column", "row"]} w="100%">
+      <Stack alignItems="center" direction="column" spacing={0}>
+        {steps?.map((step) => (
+          <React.Fragment key={step.number}>
+            <StepperCircleIcon
+              boxSize={8}
+              stroke={activeStep === step.number ? "secondary.500" : "inherit"}
+            />
+            {step.number !== steps.length ? (
```

Comment:
very nit: could just do `step.number !== steps.length && <VerticalLineIcon boxSize={20} />` instead of the ternary

Reply:
@allisonking True, but I do it this way because of  [https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx](this).
Feel free to push back though. Curious what your thoughts are.

Reply:
aha! thanks for linking, very nice article 👍